app/queries/app/servers.ts

```diff
@@ -56,3 +60,34 @@ export const queryServerName = async (appDatabase: Database, serverUrl: string)
         return serverUrl;
     }
 };
+
+export const areAllServersSupported = async () => {
+    let appDatabase;
+    try {
+        const databaseAndOperator = DatabaseManager.getAppDatabaseAndOperator();
+        appDatabase = databaseAndOperator.database;
+    } catch {
+        return false;
+    }
+
+    const servers = await queryAllServers(appDatabase);
+    for (const s of servers) {
```

Comment:
```suggestion
    for await (const s of servers) {
```
Smart adding this check